bin/build-dev

```diff
@@ -18,8 +18,8 @@ docker tag civiform-dev:latest public.ecr.aws/t1q6b4h2/civiform-dev:latest
 docker push public.ecr.aws/t1q6b4h2/civiform-dev:latest
 
 # log the docker CLI into the Docker Hub container registry
```

Comment:
Add a header comment that this is a Work In Progress and maybe why so, such that random eyes have some understanding :)

Reply:
done